src/app.js

```diff
@@ -1 +1,31 @@
 'use strict';
+
+const readline = require('readline');
+const fs = require('fs');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function createFile() {
+  terminal.question('Enter the name\n', (fileName) => {
+    if (!fileName) {
+      createFile();
+    } else {
+      terminal.question('\nEnter content\n', (content) => {
+        fs.writeFile(`./src/${fileName}.txt`, content, (error) => {
+          if (!error) {
+            terminal.write('Success');
+            terminal.close();
+          } else {
+            terminal.write('Error');
+            terminal.close();
```

Comment:
Try to avoid nesting if else. You can use ternary operator here. Your logic is the same in both cases, only the message changes